app/javascript/src/Navigation/components/header/Brand.jsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react'
+// @flow
+import * as React from 'react'
```

Comment:
Not really important but I think there's no need to import all. This is done in Flow only to be able to use React types.